src/main.rs

```diff
@@ -183,23 +183,20 @@ async fn discord() -> Result<()> {
 
     #[allow(unused_mut)]
     #[cfg(any(target_os = "linux", target_os = "windows"))]
-    let mut _webview: Option<wry::webview::WebView> = {
+    let mut _webview: wry::webview::WebView = {
         #[cfg(target_os = "windows")]
         let user_agent: String = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36".to_string();
         
         #[cfg(target_os = "linux")]
         let user_agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36".to_string();
-        
-        Some(
```

Comment:
I'd rather have the Some constructor in there

Reply:
Is there a reason for this?
It's just a bit annoying. 
I'll add it back, Sorry.